src/contracts/pods/EigenPod.sol

```diff
@@ -285,13 +299,30 @@ contract EigenPod is IEigenPod, Initializable, ReentrancyGuardUpgradeable, Eigen
             proofs.balanceRoot
         );
 
-        // mark the ETH validator as overcommitted
-        validatorStatus[validatorIndex] = VALIDATOR_STATUS.OVERCOMMITTED;
+        uint64 currentEffectiveRestakedBalanceGwei = _validatorPubkeyHashToInfo[validatorPubkeyHash].restakedBalanceGwei;
 
-        emit ValidatorOvercommitted(validatorIndex);
+        // calculate the effective (pessimistic) restaked balance
+        uint64 newEffectiveRestakedBalanceGwei = _effectiveRestakedBalanceGwei(validatorCurrentBalanceGwei);
```

Comment:
should name this function an action (i.e. `_calculateEffectedRestakedBalanceGwei`)